src/vstest.console/Processors/Utilities/StringUtilities.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Move this class to Testplatfom.Utilities

Reply:
Done

Reply:
done